library/rendering/entity_models/src/main/java/org/quiltmc/qsl/rendering/entity_models/mixin/ModelPartDataAccessor.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.rendering.entity_models.mixin;
+
+import net.minecraft.client.model.ModelCuboidData;
+import net.minecraft.client.model.ModelPartData;
+import net.minecraft.client.model.ModelTransform;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+import java.util.List;
+import java.util.Map;
+
+@Mixin(ModelPartData.class)
+public interface ModelPartDataAccessor {
+    @Accessor("cuboidData")
+    List<ModelCuboidData> cuboids();
+
+    @Accessor("rotationData")
+    ModelTransform transform();
+
+    @Accessor("children")
+    Map<String, ModelPartData> children();
+
+    @Invoker("<init>")
+    static ModelPartData create(List<ModelCuboidData> cuboids, ModelTransform rotation) {
+        throw new AssertionError("mixin broke");
```

Comment:
Maybe replace some of these with `"Unreachable"`

Reply:
I would put `throw new UnsupportedOperationException("Mixin injection failed.");` as it's already an exception used in various places of QSL.

Reply:
Yeah, that probably should be done